config/config.py

```diff
@@ -50,3 +50,6 @@
     'pool_recycle':  3600,
     'encoding':      'utf-8'
 }
+
+# where generated ansible inventory files are stored
+inventory_root_path = "/tmp"
```

Comment:
for a prod configuration we can't have /tmp/ because that is not guaranteed on every system we would need to have something here that is different provided by Python that ensures us that it is a temporary file that will be available.

Or! just use a new temporary file every time there is a call that needs it